### DISCUSSION TITLE
Can not to access the result of query when extends allOperations

### DISCUSSION THREAD

**Discussion by meotimdihia** [prisma/prisma#20329] [Q&A] [answered]
I read documentation and examples, there is nothing like this. Someone help me to solve this problem, please.

```ts
const xprisma = prisma
  .$extends({
    query: {
      md_chapters: {
        async $allOperations({ args, query, operation, model }) {
          
          if (operation != 'create' && operation != 'update') {
            return await query(args)
          }
          const cur = await query(args)
          cur.status // <-- Typescript error
```
![image](https://github.com/prisma/prisma/assets/300961/ed92e62d-5900-4672-968d-0d10160c89f2)
![image](https://github.com/prisma/prisma/assets/300961/cbcabf41-ae83-4884-a479-a31aa5e9ce48)

**Comment by ludralph** [comment, marked as answer]
Hi @meotimdihia 👋 

Is this related to the GitHub issue [#20395](https://github.com/prisma/prisma/issues/20395) you created?

**Reply by meotimdihia** [reply]
@ludralph Yes, it is a duplication. We can close this discussion 😀
